unit/generate_unit.go

```diff
@@ -250,29 +251,267 @@ var Units = []Unit{
+		Dimensions: []Dimension{
+			{Name: TimeName, Power: -1},
+		},
+	},
+	{
+		Name:        "Force",
+		Receiver:    "f",
+		PrintString: "N",
+		Suffix:      "newton",
+		Singular:    "Newton",
+		TypeComment: "Force represents a force in Newtons",
+		Dimensions: []Dimension{
+			{Name: LengthName, Power: 1},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -2},
+		},
+		ErForm: "Forcer",
+	},
+	{
+		Name:        "Inductance",
+		Receiver:    "i",
+		PrintString: "H",
+		Suffix:      "henry",
+		Singular:    "Henry",
+		TypeComment: "Inductance represents an electrical inductance in Henry",
+		Dimensions: []Dimension{
+			{Name: CurrentName, Power: -1},
+			{Name: LengthName, Power: 2},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -2},
+		},
+		ErForm: "Inductancer",
+	},
+	{
+		Name:        "Power",
+		Receiver:    "pw",
+		PrintString: "W",
+		Suffix:      "watt",
+		Singular:    "Watt",
+		TypeComment: "Power represents a power in Watts",
+		Dimensions: []Dimension{
+			{Name: LengthName, Power: 2},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -3},
+		},
+	},
+	{
+		Name:        "Resistance",
+		Receiver:    "r",
+		PrintString: "Ω",
+		Suffix:      "ohm",
+		Singular:    "Ohm",
+		TypeComment: "Resistance represents an electrical resistance, impedance or reactance in Ohms",
+		Dimensions: []Dimension{
+			{Name: CurrentName, Power: -2},
+			{Name: LengthName, Power: 2},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -3},
+		},
+		ErForm: "Resistancer",
+	},
+	{
+		Name:        "MagneticFlux",
+		Receiver:    "m",
+		PrintString: "Wb",
+		Suffix:      "weber",
+		Singular:    "Weber",
+		TypeComment: "MagneticFlux represents a magnetic flux in Weber",
+		Dimensions: []Dimension{
+			{Name: CurrentName, Power: -1},
+			{Name: LengthName, Power: 2},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -2},
+		},
+	},
+	{
+		Name:        "MagneticFluxDensity",
+		Receiver:    "m",
+		PrintString: "T",
+		Suffix:      "tesla",
+		Singular:    "Tesla",
+		TypeComment: "MagneticFluxDensity represents a magnetic flux density in Tesla",
+		Dimensions: []Dimension{
+			{Name: CurrentName, Power: -1},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -2},
+		},
+	},
+	{
+		Name:        "Pressure",
+		Receiver:    "pr",
+		PrintString: "Pa",
+		Suffix:      "pascal",
+		Singular:    "Pascal",
+		TypeComment: "Pressure represents a pressure in Pascals",
+		Dimensions: []Dimension{
+			{Name: LengthName, Power: -1},
+			{Name: MassName, Power: 1},
+			{Name: TimeName, Power: -2},
+		},
+		ErForm: "Pressurer",
+	},
+	{
+		Name:        "Velocity",
+		Receiver:    "v",
+		PrintString: "m s^-1",
+		TypeComment: "Velocity represents a velocity in meters per second",
+		Dimensions: []Dimension{
+			{Name: LengthName, Power: 1},
+			{Name: TimeName, Power: -1},
+		},
+	},
+	{
+		Name:        "Voltage",
+		Receiver:    "v",
+		PrintString: "Pa",
```

Comment:
Volts, not pascals

Reply:
I'm inventing a new system of unified units.

Reply:
... but fixed.

Reply:
Go is about making things simpler, afterall. I would probably have tried to start with Candela, but we can always get rid of that later.